board/safety/safety_subaru.h

```diff
@@ -6,11 +6,13 @@ const uint32_t SUBARU_RT_INTERVAL = 250000;    // 250ms between real time checks
 const int SUBARU_MAX_RATE_UP = 50;
 const int SUBARU_MAX_RATE_DOWN = 70;
 const int SUBARU_DRIVER_TORQUE_ALLOWANCE = 60;
+const int SUBARU_L_DRIVER_TORQUE_ALLOWANCE = 600;
 const int SUBARU_DRIVER_TORQUE_FACTOR = 10;
+const int SUBARU_L_DRIVER_TORQUE_FACTOR = 1;
```

Comment:
Can we also separate the legacy constants into a separate block? same with the msg check structs